<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cardTocard</class>
 <widget class="QDialog" name="cardTocard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">*{
	font-family:century gothic;
	font-size:24px;
}
QPushButton
{
background-color:pink;
border-radius:20px;
}
QPushButton:hover
{
background : #49ebff
}
QLineEdit
{
background-color:#333;
color:white;
}</string>
     </property>
     <property name="title">
      <string>transferation</string>
     </property>
     <widget class="QPushButton" name="pushButton_back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>687</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_date">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>420</y>
        <width>611</width>
        <height>81</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cvv2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>611</width>
        <height>81</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>CVV2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>611</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>card number to transfer money to</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>680</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>make transfer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_stablePass">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>590</y>
        <width>611</width>
        <height>71</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Permanent password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tmppassword">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>510</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#lineEdit_tmppassword
{
background-color:#333;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushB_get_password">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>510</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>get password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_amount">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>611</width>
        <height>81</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>amount</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_card_number">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>611</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>your card number</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
